vars/processTemplate.groovy

```diff
@@ -44,3 +50,21 @@ def applyDefaults(provided=[:], templateParams) {
   setParam('RELEASE_VERSION') { Utils.shWithOutput(this, "git rev-list --count HEAD") }
   return params
 }
+
+ boolean isValidateTemlate(parameters=[[:]]){
+  def pass = true
+  def mandatParamList=["RELEASE_VERSION","SOURCE_REPOSITORY_URL","SOURCE_REPOSITORY_REF"]
```

Comment:
why `SOURCE_REPOSITORY_URL` and `SOURCE_REPOSITORY_REF` are mandatory?

Reply:
If someone will not specify this, then we are evaluating that by executing git commands, and that can be done. I don't understand why we are failing the job if not provided. For RELEASE_VERSION we can ask the user to provide.

Reply:
Ref is fine, at the end it could build from the master head(still need to confirm it), what about git URL? 

We can evaluate all these details from git command, but how would `build` pass these parameters to BC definition?